BLSQ/iaso
# POLIO-1575: User testing correction

- prevent from blocking round save if no age
- make age type mandatory if either age min or age max has value

Explain what problem this PR is resolving

Related JIRA tickets : POLIO-1575

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- prevent from blocking round save if no age
- make age type mandatory if either age min or age max has value

## How to test

- create new polio campaign
- Add a round and save
- Add a second round: save should not be blocked if start and end date are set
- Add age min or age max: age type field should be in error if it has no value

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/90a43fcb-53b1-4e11-a9c7-a2c8a1851a43
